components/icons/like.tsx

```diff
@@ -0,0 +1,23 @@
+interface LikeProps {
+  width?: number
+  height?: number
+}
+
+export default function Like({ width, height }: LikeProps) {
+  return (
+    <svg
+      width={width ? width : 24}
+      height={height ? height : 20}
```

Comment:
```suggestion
export default function Like({ width = 24, height = 20}: LikeProps) {
  return (
    <svg
      width={width}
      height={height}
```

요렇게 default 값 선언해서, 렌더링 부분에는 조건없이 변수만 전달해도 좋아보여요